config/custom-environment-variables.yaml

```diff
@@ -24,6 +24,12 @@ builds:
     # Upper limit on incoming uploads to builds
     maxByteSize: BUILDS_MAX_BYTES
 
+commands:
+    # How long should commands stay around for
+    expiresInSec: COMMANDS_EXPIRE_TIME
```

Comment:
If different values are set in `BUILDS_EXPIRE_TIME` and `COMMANDS_EXPIRE_TIME`, is there any problem?

Reply:
I had a misunderstanding. Please ignore this comment :bow: